e2e/.docker/.buildNodeID

```diff
@@ -0,0 +1 @@
+8162aa64cf9962b818007cfe7bb9ce9c4b14e7e5308c12847134c6574797e2c6
```

Comment:
What is this file for? 🤔

Reply:
don't know. Possibly created by accident as part of a failed test. I will remove it

Reply:
That would be my guess. It comes from https://github.com/docker/cli/blob/master/cli/command/image/build_session.go#L142